### ISSUE TITLE
conditionals: allow keys in then/else of assignment

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#522]
```
[powersaving/gps]
assign/condition = (device != 'vehicle') ?  (battery/level) : ('full')
```

leads to

```
Using name user/example/highlevel/powersaving/gps
Set string to full
The command set failed while accessing the key database with the info:
Error (#134) occurred!
Description: Syntax error
Ingroup: plugin
Module: conditionals
At: /home/jenkins/workspace/workspace/elektra-git-buildpackage-jessie/libelektra/src/plugins/conditionals/conditionals.c:376
Reason: Invalid syntax: "battery/level". Check kdb info conditionals for additional information

Mountpoint: user/example/highlevel
Configfile: /home/markus/.config/highlevel.ini.8639:1455571677.214324.tmp
```

expected: `battery/level` should be a key to lookup

**Comment by tom-wa** [comment]
@markus fixed in #540

**Comment by markus2330** [comment]
Thank you, works fine now!
